assets/openshift-apiserver/config.yaml

```diff
@@ -22,7 +22,7 @@ auditConfig:
   maximumFileSizeMegabytes: 100
   maximumRetainedFiles: 10
   policyConfiguration:
-    apiVersion: audit.k8s.io/v1beta1
+    apiVersion: audit.k8s.io/v1
```

Comment:
There are other references to the `audit.k8s.io/v1beta1` API that need to be changed.  You will also need to generate the bindata.

Reply:
Sounds good. I will change the following files to include ```audit.k8s.io/v1```:
1. [bindata.go](https://github.com/openshift/ibm-roks-toolkit/blob/master/pkg/assets/bindata.go)
2. [audit-policy.yaml](https://github.com/openshift/ibm-roks-toolkit/blob/master/assets/oauth-apiserver/audit-policy.yaml) located in assets/oauth-apiserver
3. [default-audit-policy.yaml](https://github.com/openshift/ibm-roks-toolkit/blob/master/assets/kube-apiserver/default-audit-policy.yaml) located in assets/kube-apiserver